app/Models/IncidentsHistory.php

```diff
@@ -14,6 +14,10 @@
 use Illuminate\Database\Eloquent\SoftDeletes;
 use CachetHQ\Cachet\Presenters\IncidentsHistoryPresenter;
 
+/**
+ * This is a model for the database table incidents_histories.
```

Comment:
... incidents_histories, which tracks ...
Or start your second sentence like: It tracks ...